### PR TITLE
SCP-2656 - Update examples for prototype

### PR DESCRIPTION
**Pull request by palas**
This PR:
- Splits description into "short description" and "long description"
- Updates examples to use and match the reviewed metadata (for Escrow, ZCB, and CFD with and without oracle)

Deployed to:
- https://pablo.marlowe.iohkdev.io/
- https://pablo.marlowe-dash.iohkdev.io/

Pre-submit checklist:
- Branch
    - [x] Tests are provided (if possible)
    - [x] Commit sequence broadly makes sense
    - [x] Key commits have useful messages
    - [x] Relevant tickets are mentioned in commit messages
    - [x] Formatting, materialized Nix files, PNG optimization, etc. are updated
- PR
    - [x] Self-reviewed the diff
    - [x] Useful pull request description
    - [x] Reviewer requested

### CODE REVIEW

File: marlowe-dashboard-client/src/Template/View.purs [hunk 1]
```diff
@@ -144,7 +144,8 @@ contractOverview contractTemplate =
             [ contractIcon contractTemplate.metaData.contractType
             , text $ contractTemplate.metaData.contractName <> " overview"
             ]
-        , p_ $ markdownToHTML contractTemplate.metaData.contractDescription
+        , p_ $ markdownToHTML contractTemplate.metaData.contractShortDescription
```
Comment (merivale) [inline marlowe-dashboard-client/src/Template/View.purs]:
```suggestion
        , p [ classNames [ "mb-4" ] ] $ markdownToHTML contractTemplate.metaData.contractShortDescription
```

**Review by merivale — APPROVED** [review]
LGTM. A few tweaks were still needed for Marlowe Run. I was going to write comments, but then thought it was easier just to push a commit on top. Hope you don't mind.

File: marlowe-dashboard-client/src/Template/View.purs (hunk 1 above)
Comment (palas) [reply marlowe-dashboard-client/src/Template/View.purs]: That is great, thanks. I didn't know the suggested change feature too, nice :)

**Comment by hrajchert** [comment]
LGTM, we probably need to remove other contracts from marlowe run and to preprocess markdown

Screen Shot 2021-08-20 at 12 01 03

**Review by hrajchert — APPROVED** [review]

**Pull request merged**